pkg/trace/api/payload.go

```diff
@@ -29,6 +29,9 @@ type Payload struct {
 
 	// ClientDroppedP0s specifies the number of P0 traces chunks dropped by the client.
 	ClientDroppedP0s int64
+
+	// ProcessTags is a list of tags describing an instrumented process.
+	ProcessTags string
```

Comment:
Process Tags are already passed within the `TracerPayload.Tags["_dd.tags.process"]`, why have this dedicated field with the same data?

Reply:
it's used to pass it to stats without doing a map access, agreed though it makes a dupe, I'll move stats to pick it from this field